shared/cfg/guest-os/Linux/RHEL/5.11/i386.cfg

```diff
@@ -7,7 +7,7 @@
         kernel = images/rhel511-32/vmlinuz
         initrd = images/rhel511-32/initrd.img
     unattended_install.cdrom, check_block_size.4096_512, check_block_size.512_512, svirt_install:
-        cdrom_cd1 = isos/linux/RHEL-5.10-i386-DVD.iso
+        cdrom_cd1 = isos/linux/RHEL-5.11-i386-DVD.iso
```

Comment:
The hash shouldn't reflect the file name, only the file content should affect the hash.